### DISCUSSION TITLE
helm upgrade --install dagster dagster/dagster -f .\values.yaml Release "dagster" does not exist. Installing it now. Error: chart requires kubeVersion: >= 1.15.0-0 which is incompatible with Kubernetes v1.11.0+d4cacc0

### DISCUSSION THREAD

**Discussion by slamer59** [dagster-io/dagster#4874] [Q&A] [answered]
Hello,
I tried to install dagster from this Chart on Openshift and got this anwser. 

I don't know what error is this ? 
Is it related to the version of Openshift ? I think I point to an old openshift version.

Is there any workaround ?

Regards

**Comment by rexledesma** [comment, marked as answer]
Your cluster must be on Kubernetes version 1.15 if you want to install our chart.

**Reply by slamer59** [reply]
Thanks for this confirmation. So do you think it s feasible to construct a chart with an mulder version ?
I dont see blocker.
